app/services/hyrax/workflow/status_list_service.rb

```diff
@@ -38,7 +41,10 @@ def search_solr
           actionable_roles = roles_for_user
           logger.debug("Actionable roles for #{user.user_key} are #{actionable_roles}")
           return [] if actionable_roles.empty?
-          WorkRelation.new.search_with_conditions(query(actionable_roles), rows: 1000)
+          work_query = WorksSearchBuilder.new([:filter_models], self).query
+          work_query["fq"] += query(actionable_roles)
```

Comment:
I couldn't get the query to look right when using `WorksSearchBuilder.with(fq: query(actionable_roles).rows(1000)`.